### PR TITLE
Add pipeline status widget to .md

### PR DESCRIPTION
**Pull request by MenelvagorMilsom**
Overview
---
- Adding status widget for Azure Pipelines CI

### CODE REVIEW

**Comment by MenelvagorMilsom** [comment]
Presently, continuous integration is disabled for this pipeline, but I can enable it as soon as required.

File: docfx.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
```
Comment (paco-ms) [inline docfx.json]: You probably didn't mean to include this file as part of the PR ?
```suggestion
{remove this file}
```

Comment (MenelvagorMilsom) [reply docfx.json]: Yeah, this file was given to me by Bernadette. She said that it should be added to the MRTK root dir.

Comment (paco-ms) [reply docfx.json]: Ok. If we (reasonably) can we should try to separate the commits for the azure pipeline and those for generating documentation.

**Comment by paco-ms** [comment]
Let's hold on this commit until we move the build pipeline into a public project. We want users to be able to inspect build results.

**Comment by davidkline-ms** [comment]
Marking as do not merge until the pipeline changes are complete

**Comment by MenelvagorMilsom** [comment]
@keveleigh @davidkline-ms I've updated the status badge to point to the newer build pipeline. This build pipeline just had it's visibility set to public so we should be ready to merge this PR.

**Pull request merged**